inst

```diff
@@ -245,19 +268,23 @@ nextcloud_ensure_extended_attributes () {
         --set valueRequired='0' \
         --set CLIName='nextcloudEnabled' \
         --set syntax='boolean' \
-        --set default="1" \
-        --set tabAdvanced='1' \
+        --set default="0" \
+        --set tabAdvanced='0' \
```

Comment:
In the original join script the parameters `default` and `tabAdvanced` had been set to 1 and now are changed to 0. What is the difference in the semantic? I assume the intended behavior is that new created users have access to Nextcloud by default and this is controlled with `default`. This behavior should not change.

Reply:
Sorry, `default=0` was a typo. This must be `default=1`. The default value is now controlled via the settings/extended_options. The nextCloudActive attribute is in the future invisible and will be automatically set to true.

tabAdvanced changes indeed, because the settings shouldn't be shown anymore in the `Advances settings` tab but on a own tab for the app.

Reply:
I forced pushed a new version.